content/documentation/getting-started/index.adoc

```diff
@@ -299,6 +299,31 @@ The login option can be specified in the Kiali CR when installing Kiali. For ins
 auth:
   strategy: openshift
 ----
+*ldap*: This option allows you to set the authentication strategy so that the user is validated using the LDAP server, then an authentication token is generated. A sample configuration for LDAP is given below
+----
+auth:
+  strategy: ldap
+  ldap:
+    # AD IP host and port
```

Comment:
See my comments in the related Kiali PR.  "# AD IP host and port" comment - the AD is confusing (what if I'm not using Active Directory? Is that what AD means? Does this LDAP feature only support AD? etc etc.) ... It also is obvious what host and port are. So that comment can be removed.
```suggestion
```